godot-core/src/builtin/projection.rs

```diff
@@ -0,0 +1,443 @@
+    ///
+    /// _Godot equivalent: Projection.get_fov()_
+    pub fn fov(&self) -> f64 {
+        self.as_inner().get_fov()
+    }
+
+    /// Returns the vertical field of view of a projection (in degrees) which
+    /// has the given horizontal field of view (in degrees) and aspect ratio.
+    ///
+    /// _Godot equivalent: Projection.get_fovy()_
+    pub fn fov_y_of(fov_x: f64, aspect: f64) -> f64 {
+        InnerProjection::get_fovy(fov_x, aspect)
+    }
+
+    /// Returns the factor by which the visible level of detail is scaled by
+    /// this Projection.
+    ///
+    /// _Godot equivalent: Projection.get_lod_multiplier()_
+    pub fn lod_multiplier(&self) -> f64 {
+        self.as_inner().get_lod_multiplier()
+    }
+
+    /// Returns the number of pixels with the given pixel width displayed per
+    /// meter, after this Projection is applied.
+    ///
+    /// _Godot equivalent: Projection.get_pixels_per_meter()_
+    pub fn pixels_per_meter(&self, pixel_width: i64) -> i64 {
+        self.as_inner().get_pixels_per_meter(pixel_width)
+    }
+
+    /// Returns the clipping plane of this Projection whose index is given by
+    /// plane.
+    ///
+    /// _Godot equivalent: Projection.get_projection_plane()_
+    pub fn projection_plane(&self, plane: ProjectionPlane) -> Plane {
+        self.as_inner().get_projection_plane(plane as i64)
+    }
+
+    /// Returns the dimensions of the viewport plane that this Projection
+    /// projects positions onto, divided by two.
+    ///
+    /// _Godot equivalent: Projection.get_viewport_half_extents()_
+    pub fn viewport_half_extents(&self) -> Vector2 {
+        self.as_inner().get_viewport_half_extents()
+    }
+
+    /// Returns the distance for this Projection beyond which positions are
+    /// clipped.
+    ///
+    /// _Godot equivalent: Projection.get_z_far()_
+    pub fn z_far(&self) -> f64 {
+        self.as_inner().get_z_far()
+    }
+
+    /// Returns the distance for this Projection before which positions are
+    /// clipped.
+    ///
+    /// _Godot equivalent: Projection.get_z_near()_
+    pub fn z_near(&self) -> f64 {
+        self.as_inner().get_z_near()
+    }
+
+    /// Returns a Projection that performs the inverse of this Projection's
+    /// projective transformation.
+    ///
+    /// _Godot equivalent: Projection.inverse()_
+    pub fn inverse(self) -> Self {
+        self.glam(|mat| mat.inverse())
+    }
+
+    /// Returns `true` if this Projection performs an orthogonal projection.
+    ///
+    /// _Godot equivalent: Projection.is_orthogonal()_
+    pub fn is_orthogonal(&self) -> bool {
+        self.as_inner().is_orthogonal()
+    }
+
+    /// Returns a Projection with the X and Y values from the given [`Vector2`]
+    /// added to the first and second values of the final column respectively.
+    ///
+    /// _Godot equivalent: Projection.jitter_offseted()_
+    pub fn jitter_offseted(&self, offset: Vector2) -> Self {
```

Comment:
"offseted" is not a real word, the past of "offset" is "offset".
I understand why they used it though...

Maybe `jitter_offset()` + `#[must_use]`?

Reply:
it's what godot uses, but yeah that makes sense.

Reply:
I reported it with Godot devs, and while they agreed it should be changed, it was too late. Godot 4.0 is supposed to be released today 😉 

Another option would be `jitter_displaced()`.